cli/src/lib.rs

```diff
@@ -649,6 +649,20 @@ mod tests {
             base
         }
 
+        fn config_to_toml_value(config: PartialUserConfig) -> Result<toml::Value> {
+            use iroha_crypto::ExposedPrivateKey;
+            let private_key = config.private_key.as_ref().unwrap().clone();
+            let genesis_private_key = config.genesis.private_key.as_ref().unwrap().clone();
+            let mut result = toml::Value::try_from(config)?;
+
+            // private key will be serialized as "[REDACTED]" so need to restore it
+            result["private_key"] = toml::Value::try_from(ExposedPrivateKey(private_key))?;
```

Comment:
It's unfortunate that this will require us to remember to update this function if we ever move/add new private keys to the config, but I didn't think of any alternative other than making `PartialUserConfig` serialize with exposed keys, which is probably not what we want.

Reply:
Yes, but this function is used only in tests, so in case new private keys added to the config, tests will fail